articles/integrations/azure-api-management/_stepnav.html

```diff
@@ -0,0 +1,14 @@
+<div class="tutorial-prev-next" data-swiftype-index="false">
+  <% if (typeof prev !== 'undefined') { %>
+  <div class="tutorial-prev-next-prev">
+    <div class="tutorial-prev-next-header">Previous Tutorial</div>
+    <a href="${prev[1]}"><i class="icon-budicon-463"></i> ${prev[0]}</a>
+  </div>
+  <% } %>
+  <% if (typeof next !== 'undefined') { %>
+  <div class="tutorial-prev-next-next">
```

Comment:
Since this is one tutorial (with multiple steps) I'm thinking that a simple `Next` would be better than `Next Tutorial`. I know we use `Next Tutorial` also in our quickstarts and you probably used this because of that, but `Next` is simpler (and therefore better).